agr_literature_service/lit_processing/utils/okta_utils.py

```diff
@@ -60,9 +62,14 @@ def get_authentication_token():  # pragma: no cover
     okta_file = base_path + 'okta_token'
     token = ''
     if path.isfile(okta_file):
-        with open(okta_file, 'r') as okta_fh:
-            token = okta_fh.read().replace("\n", "")
-            okta_fh.close
+        one_day_ago = datetime.now() - relativedelta(days=1)
```

Comment:
1 day is probably fine.  If okta is really strict about 1 day, we might have issues if there's some lag somewhere, but hopefully it'll be fine, and we don't currently have any processes that take 24+ hours

Reply:
Yes, you are right! hopefully everything will work out fine. If not, we can fix again. =)